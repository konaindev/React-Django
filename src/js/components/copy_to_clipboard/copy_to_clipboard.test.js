import renderer from "react-test-renderer";

import CopyToClipboard from "./index";

describe("CopyToClipboard", () => {
  it("renders enabled button correctly", () => {
    const tree = renderer
      .create(<CopyToClipboard textToCopy="Some Text" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("renders disabled button correctly", () => {
    const tree = renderer
      .create(<CopyToClipboard disabled textToCopy="Some Text" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
