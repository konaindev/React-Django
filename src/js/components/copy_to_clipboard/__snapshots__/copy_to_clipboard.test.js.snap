// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CopyToClipboard renders disabled button correctly 1`] = `
<button
  className="button button--disabled button--default copy-to-clipboard"
  onClick={[Function]}
>
  Copy Text
</button>
`;

exports[`CopyToClipboard renders enabled button correctly 1`] = `
<button
  className="button button--primary copy-to-clipboard"
  onClick={[Function]}
>
  Copy Text
</button>
`;
